Automattic/newspack-popups
# fix: prevent duplicated pageviews if amp-analytics was loaded externally

### All Submissions:






### Changes proposed in this Pull Request:



Fixes as doubled-pageview issue appearing when `amp-analytics` was at the same time loaded and polyfilled, effectively running twice. 

### How to test the changes in this Pull Request:

1. In a non-AMP post or page, add the following snippet to load `amp-analytics`:

```html
<script src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
```

3. On `master`, visit the post or page
4. In the network tab, filter by `t=pageview`, observe two requests
5. Switch to this branch, rebuild, observe only one page view request is sent

### Other information:
